Recrutify/Recrutify.Services/Services/ScheduleService.cs

```diff
@@ -47,5 +56,25 @@ public async Task<ScheduleDTO> GetByDatePeriodForCurrentUserAsync(DateTime? date
             var schedules = await _scheduleRepository.GetByDatePeriodAsync(userId, date.Value, daysNum);
             return _mapper.Map<ScheduleDTO>(schedules);
         }
+
+        public async Task UpdateScheduleSlotsForCurrentUserAsync(IEnumerable<DateTime> dates)
+        {
+            var currentUserId = _userProvider.GetUserId();
+            var scheduleSlotByCurrentUser = await _scheduleRepository.GetScheduleSlotsByDatePeriodAsync(currentUserId, dates.Min());
```

Comment:
If user min date is Friday, is it necessary to get from data base next 7 days slots ?

Reply:
Or will you agree with FE that they always pass you all week slots ? When, you should validate length of the array they pass

Reply:
+